docs/forgedev/index.md

```diff
@@ -62,11 +62,22 @@ If you're more of a visual person, cpw has uploaded [a video](https://www.youtub
 !!! Note
     These steps will only work reliably from IDEA version 2016 onwards. Older versions didn't have the appropriate Gradle support and did not support Forge development workspaces.
 
+#### IDEA 2019
+
+ 1. Import Forge's `build.gradle` as an IDEA project. For this, simply click `Import Project` from the `Welcome to IntelliJ IDEA` splash screen, then select the `build.gradle` file.
+1. After IDEA is done importing the project and indexing the files, run the Gradle setup task.  Either:
+   1. open the Gradle sidebar on the right hand side of your screen, then open the `forge` project tree, select `Tasks`, then `other` and double-click the `setup` task (may also appear as `MinecraftForge[Setup]`.  Or alternatively:
+   1. tap the CTRL key twice, and type `gradle setup` in the `Run` command window that pops up.
+
+You can then run Forge using the `forge_client` gradle task (`Tasks -> fg_runs -> forge_client`): right-click the task and select either `Run` or `Debug` as desired.
+
+#### IDEA older versions
```

Comment:
You should add a disclamer to the top that "only 2019 and newer versions should be used"